backend/src/routers/course.js

```diff
@@ -46,7 +46,7 @@ router.get('/api/course/search', logRequest, async (req, res) => {
                 dept, level, num: { $regex: num, $options: "i" }
             });
         }
-
+        // should send statuscode.OK here too
```

Comment:
We need to send a success code for the successful case.

Reply:
@anthopark what do you think?

Reply:
@emmohac Yes, there should be sending statuscode.OK. thanks!

Reply:
OK I can work on that